### ISSUE TITLE
[misc] S

### ISSUE THREAD

**Issue by nils-schween** [djcb/mu#2186]
**Note**: for questions, please use the mailing-list: https://groups.google.com/g/mu-discuss

**Describe the issue**
A clear and concise description, i.e. what you expected/desired to happen and what actually happened.

**Environment**
If applicable, please describe the versions of OS, Emacs, mu etc. you are using.

**Comment by djcb** [comment]
Can you attach a message where this happens? Thanks.

**Comment by nils-schween** [comment]
Yes, of course. The file has the suffix ".md", because github only allows files with specific suffixes.
[mail_without_x_flag.md](https://github.com/djcb/mu/files/7656346/mail_without_x_flag.md)
 
And thanks for investigating this.

**Comment by djcb** [comment]
Ah, okay... I pushed a change for this. Does it help?
